doc/sphinx/make-your-own-compositor.md

```diff
@@ -0,0 +1,258 @@
+---
+discourse: 4911,5164,5603,6756,8037
+---
+
+# Using Mir Libraries to Build Your Own Wayland Compositor
+If you've seen [the demo tutorial](explore-mir-using-demos), you're probably
+taken away by Mir's capabilities and want to use it and the supporting
+libraries to build your own Wayland compositor. This tutorial will guide you
+through the installation of dependencies, writing a simple compositor,
+building it, and running it. You'll be even more impressed by how easy it is to
+create a Wayland compositor using Mir!
+
+## Assumptions
+This tutorial assumes that:
+1. The reader is familiar with C++ and CMake.
+2. The reader has a C++ compiler with C++ 23 support.
+
+## A barebones Mir compositor
+This section will cover the needed dependencies and how to install them, the
+minimum code needed for a Mir compositor, how to build this code, and finally
+how to run your compositor.
+
+### Dependencies
+Before you start coding, you'll need to install `libmiral`, and
+`mir-graphics-drivers-desktop` which can be done on different distros as
+follows:
```

Comment:
There are additional "build" dependencies: c++, cmake, etc.

Reply:
I think it's a bit too much to put that dependencies section. Would the assumptions section work? (see latest commit)